pkg/api/api.proto

```diff
@@ -25,17 +25,17 @@ option go_package = "github.com/containerd/nri/pkg/api;api";
 // only covers plugin registration and unsolicited container updates.
 // The rest of the API is defined by the Plugin service.
 service Runtime {
-    // RegisterPlugin registers the plugin with the runtime.
-    rpc RegisterPlugin(RegisterPluginRequest) returns (Empty);
-    // UpdateContainers requests unsolicited updates to a set of containers.
-    rpc UpdateContainers(UpdateContainersRequest) returns (UpdateContainersResponse);
```

Comment:
nit: can we get rid of these unrelated changes ? AFAICT, they just change tabulation.